paycalculator/Program.cs

```diff
@@ -1,36 +1,48 @@
 ﻿using Newtonsoft.Json;
 using System;
 using Employees.Entites;
+using Employees;
 using Employees.Services;
 
 namespace paycalculator
 {
     class Program
     {
-        
 
         public static void Main(string[] args)
         {
+            PermanentEmployee permanentEmployee1 = new PermanentEmployee(name: "Joe Bloggs", 40000, 5000, holidayAllowance: 21);
```

Comment:
I'm not aware of any actual "end purpose" of this example program, but perhaps think about tidying up this method, and maybe you could implement a menu system to view each function that you want? i.e. 1. View all employees 2. Write employees etc.

Reply:
I am currently working on a switch statement that will call methods depending on the users input.